backstop_scenarios.js

```diff
@@ -78,22 +82,28 @@ try {
   scenarioPaths = defaultPaths.paths;
 }
 
-/**
- * @todo we need to allow for a deep merge so that the template-paths.js can include an override for the Home and
- * Forced 404 paths.
- */
-for (let i=0;i<scenarioPaths.length;++i) {
-  let newScenario = scenarioPaths[i];
-  let newPath = newScenario.path;
-  scenarioPaths[i].referenceUrl = `${baselineURL}${newPath}`;
-  scenarioPaths[i].url = `${testURL}${newPath}`;
+//now that we have our initial collection of paths, we need to merge any that are duplicates
+newScenarios = scenarioPaths.reduce((accumulator, currentValue, currentIndex, array) => {
```

Comment:
Are `currentIndex` and `array` used? Can't they be removed?

Reply:
yes, i think we can. I included them initially because I thought I was going to use them, but didnt end up needing them. I'll update this.